src/olympia/blocklist/cron.py

```diff
@@ -23,10 +24,7 @@ def upload_mlbf_to_kinto():
         log.info('Upload MLBF to kinto cron job disabled.')
         return
     last_generation_time = get_config(MLBF_TIME_CONFIG_KEY, 0, json_value=True)
-    if last_generation_time > _get_blocklist_last_modified_time():
-        log.info(
-            'No new/modified Blocks in database; skipping MLBF generation')
-        return
+    last_blocked_count = get_config(MLBF_COUNT_CONFIG_KEY, 0, json_value=True)
```

Comment:
This is only used once, so it could be inlined in the expression for `need_mlbf`, but it does make the code a bit more obvious to have this named variable, so I think it's fine either way.

Reply:
It got removed in the revision anyway